lib/lambda_ethereum_consensus/p2p/subscriber.ex

```diff
@@ -40,9 +40,11 @@ defmodule LambdaEthereumConsensus.P2P.Subscriber do
 
   @impl true
   def handle_info(
-        {:gossipsub, {topic, message}},
+        {:gossipsub, {topic, msg_id, message}},
         %{topic: topic, queue: queue, demand: demand} = state
       ) do
+    # TODO: validate
+    Libp2pPort.validate_message(msg_id, :accept)
```

Comment:
is it specified what a valid message means? For example, just that we can decode it or the contents as well?

What are the consequences of sending a reject or ignore? I guess that it is not going to be gossiped to other peers? Will the peer that sent this bad message be penalized somehow?

Reply:
Oops. I forgot to document the function 😋. Will do that

Reply:
Done: 449049e9a854ea13ca64ba2b22ce060377bc6063

Reply:
thanks!